pkg/auth/auth_handler.go

```diff
@@ -217,7 +219,10 @@ func (a *AuthHandler) CreateAuthenticator() func(name, in string, authenticate s
 			}
 			p, err := authenticate(token)
 			if err != nil {
-				return false, nil, err
+				if common.IsKnownError(err) {
```

Comment:
Why return true?

Reply:
Otherwise it will ignore the error and just return 401

Reply:
See here: 
https://github.com/go-openapi/runtime/blob/master/middleware/router.go#L189

The error is ignored if we return false.

Reply:
Because of https://github.com/go-openapi/runtime/blob/master/middleware/router.go#L189 
thanks @rollandf for finding it